cmd/authenticator/main.go

```diff
@@ -42,7 +38,7 @@ func main() {
 
 	err = backoff.Retry(func() error {
 		for {
-			infoLogger.Printf(log.CAKC006I, authn.Config.Username)
+			log.Info(log.CAKC006I, authn.Config.Username)
```

Comment:
shouldn't this be a debug log message? why does this need to be logged every tiime?

Reply:
well, currently there are no debug log messages in the authn-client. this PR makes changes to the client so that we have this ability but deciding which messages should be logged at each level can be noisy. we have [this issue](https://github.com/cyberark/conjur-authn-k8s-client/issues/134) for this and we can move to that once this PR is merged